app/pdfs/certificate_pdf.rb

```diff
@@ -2,47 +2,73 @@
 
 class CertificatePDF < Prawn::Document
   attr_accessor :path
+  attr_reader :certificate
 
   PDF_OPTIONS = {
     page_size:   'A5',
     page_layout: :landscape,
     margin:      [30, 75]
-  }
+  }.freeze
```

Comment:
@adilsoncarvalho Like the other change, that was made by rubocop, but I've adopting this change when I did any code, because the use of freeze prevents a reassignment of value (even if is a constant).

You could see an example here: http://stackoverflow.com/a/27702669/2649707
More content here: http://blog.honeybadger.io/when-to-use-freeze-and-frozen-in-ruby/

Adding to this context, ruby 2.2+ (if I'm not wrong) have made significant changes, that reduce the memory allocation when we use freeze content (but now I don't know if it includes other kind of objects, not only strings)